lib/picam/camera.ex

```diff
@@ -33,11 +40,19 @@ defmodule Picam.Camera do
 
   def handle_info({_, {:data, jpg}}, state) do
     Task.start(fn -> dispatch(state.requests, jpg) end)
-    {:noreply, %{state | requests: []}}
+    {:noreply, %{state | requests: [], offline: false}}
+  end
+
+  def handle_info(:reconnect_port, state) do
+    executable = Path.join(:code.priv_dir(:picam), "raspijpgs")
```

Comment:
Let's refactor the `executable` and `port` lines to a separate private function that we can call from `init` as well.

Something like:

```elixir
defp spawn_port() do
  executable = :code.priv_dir(:picam) |> Path.join("raspijpgs")
  Port.open({:spawn_executable, executable}, [{:packet, 4}, :use_stdio, :binary, :exit_status])
end
```